source/org/intellij/plugins/ceylon/ide/project/CeylonPageTwo.ceylon

```diff
@@ -300,10 +302,16 @@ shared class CeylonPageTwo()
     shared actual void load(IdeaCeylonProject config) {
         form.\imodule = config.ideArtifact;
         value systemRepository = config.ideConfiguration.systemRepository;
-        form.systemRepository.text = if (!exists systemRepository) then null else systemRepository.string;
+        //TODO: Investigate. `text` setter has annotation `@Nullable`,
```

Comment:
Ok. I wanted to make sure because of - "Inherit getter nullable if there's not annotation on setter" https://github.com/ceylon/ceylon/commit/cfb5777db9e9722166ee38a996acb1b6e1b2084e . As far as I know this feature is not yet documented.
Thanks!